test/Serialization/upgrades/setup_tests.jl

```diff
@@ -6,7 +6,7 @@ if !isdefined(Main, :serialization_upgrade_test_path) ||
   !isdir(Main.serialization_upgrade_test_path) ||
   !isfile(joinpath(Main.serialization_upgrade_test_path, "LICENSE.md"))
 
-  serialization_upgrade_test_path = let commit_hash = "a28d9e4dcc89de46377061e2f42ee09739735b97"
+  serialization_upgrade_test_path = let commit_hash = "ec819b9ddc0fc2bd0b842cfb1ea2341bd34beff4"
```

Comment:
```suggestion
  serialization_upgrade_test_path = let commit_hash = "0b7ff9e5e1135ddeaf83bd20313946183522e0ae"
```
https://github.com/oscar-system/serialization-upgrade-tests/pull/7 is merged